theano/gpuarray/tests/test_basic_ops.py

```diff
@@ -396,7 +396,7 @@ def check(dtype, N, M_=None):
         k_symb = numpy.asarray(0)
         out = T.eye(N_symb, M_symb, k_symb, dtype=dtype)
         f = theano.function([N_symb, M_symb],
-                            out,
+                            T.stack(out),
```

Comment:
Yeah, this is still needed. I tested. And yea, this is also because the input is not on GPU. Stack does not have input on GPU, but the T.stack() can give output Variables which could be on GPU, right ? And this is being used by the function that we compile. This was my understanding in doing this.